paper/ms.tex

```diff
@@ -618,9 +604,9 @@ \section{Discussion}
 Neptune-mass planets within 0.3~AU with moderately high eccentricity
 ($e\sim0.1-0.3$) \footnote{
   The two stars were monitored by \project{HARPS}
-  campaign, and it has recently been reported by \citealt{} that HD~20781 hosts
+  campaign, and it has recently been reported by \citealt{todo} that HD~20781 hosts
```

Comment:
I had to add some dummy text into `\citealt{}` for my version of texlive to build the paper... strange.